src/plugins/europeana/entity.js

```diff
@@ -92,9 +93,15 @@ export default (context = {}) => {
 
     imageUrl(entity) {
       let url = null;
-
+      // `contentfulImage` is an override used in theme chips, using contentful assets.
+      if (entity?.contentfulImage) {
```

Comment:
This is duplicating logic from src/components/generic/OptimisedImage.vue. Can't we reuse that component instead?

Reply:
It's not possible to just use that component as the component renders the image, but we just want to get a URL here.

However I've refactored the actual optimisation logic into a contentful-utils plugin so that it does not need to be duplicated between the optimisedImage component and other places where we just need the optimised image URL.

I've also removed the need for the entity component to have to manage contentful assets. Instead the contentful assets are handled separately in the relatedcollections and quicksearch components respectively.